blueprints/ember-flexberry/index.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
       '        storeDeprecationMessages: false,\n' +
       '        storePromiseErrors: true,\n' +
       '        showPromiseErrors: true,\n' +
+      '        errorMessageFilterActive: true,\n' +
```

Comment:
This will change the existing behavior after regeneration, you need to change to `false`.